src/cli-main.ts

```diff
@@ -97,6 +97,9 @@ export async function main(options: Options = {}) {
         const external = ensureArray(flags.external)
         options.external = external
       }
+      if (flags.target) {
+        options.target = flags.target.indexOf(',') >= 0 ? flags.target.split(',') : flags.target
+      }
```

Comment:
I guess you can just do `.split` without checking index

Reply:
oops I guess this is necessary otherwise this will fail: https://github.com/egoist/tsup/blob/529be6895dd0995a70f60d5a4329eb184743bb4a/src/plugins/es5.ts#L11

Reply:
Oh, I will revert previous change